staging/src/k8s.io/apimachinery/pkg/util/rand/rand.go

```diff
@@ -80,7 +80,7 @@ func Perm(n int) []int {
 const (
 	// We omit vowels from the set of available characters to reduce the chances
 	// of "bad words" being formed.
-	alphanums = "bcdfghjklmnpqrstvwxz2456789"
+	alphanums = "bcdfghjklmnpqrstvwxz1234567890"
```

Comment:
This PR does not make sense to me since the original purpose of removing digits and vowels from the set is:

```
	// We omit vowels from the set of available characters to reduce the chances
	// of "bad words" being formed.
```

Besides, I don't think that improve the performance of the random number generation could help the overall performance.

Reply:
@draveness 
I am trying to make sense out of your comment.

The comment is code only mentioned vowels. There is no reasoning why digits were excluded. If someone knows, we should enrich the comment so that people looking at this code don't have the same doubt.

The staging/src/k8s.io/apimachinery/pkg/util/rand/rand_test.go specifically included all digits for its validation assertion.

If there is no need to optimize random number generation (which I doubt), we should take out BenchmarkRandomStringGeneration.

Reply:
1 and 3 are omitted because they look like l and e